cmd/crc/cmd/version.go

```diff
@@ -1,13 +1,36 @@
+/*
+Copyright (C) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package cmd
 
 import (
+	"encoding/json"
 	"fmt"
 
 	crcPkg "github.com/code-ready/crc/pkg/crc"
+	outputPkg "github.com/code-ready/crc/pkg/crc/output"
```

Comment:
no need to add `pkg`

Reply:
You mean `outputPkg`? Usually it is good practice to add `Pkg` as we might be having similar names in `cmd` or `pkg`. I do not see a downside while using `outputPkg` as the name.

Reply:
for consistency with other code we do not add `Pkg`; alias is not needed.